WalletWasabi.Gui/Tabs/WalletManager/WalletManagerViewModel.cs

```diff
@@ -84,10 +67,39 @@ public void SelectTestPassword(string walletName)
 			passwordTestViewModel.SelectedWallet = passwordTestViewModel.Wallets.FirstOrDefault(w => w.WalletName == walletName);
 		}
 
-		public ViewModelBase CurrentView
+		public override void OnOpen(CompositeDisposable disposables)
 		{
-			get => _currentView;
-			set => this.RaiseAndSetIfChanged(ref _currentView, value);
+			base.OnOpen(disposables);
+
+			LoadWalletDesktop = new LoadWalletViewModel(this, LoadWalletType.Desktop);
+
+			Categories = new ObservableCollection<CategoryViewModel>
+			{
+				new GenerateWalletViewModel(this),
+				new RecoverWalletViewModel(this),
+				LoadWalletDesktop,
+				new LoadWalletViewModel(this, LoadWalletType.Password),
+				new ConnectHardwareWalletViewModel(this)
+			};
+
+			SelectedCategory = Categories.FirstOrDefault();
+
+			this.WhenAnyValue(x => x.SelectedCategory).Subscribe(category =>
+			{
+				category?.OnCategorySelected();
+
+				CurrentView = category;
+			});
+		}
```

Comment:
If I remove this function Codescene issue goes away. So the problem is somewhere here.

Reply:
I have no idea what is the problem.

Reply:
It is a copy/pasted code from the constructor. I cannot check the problem details it is not visible for branches. I will not care about it now, merging.